src/runtime/tests/arc-test.ts

```diff
@@ -1110,10 +1085,11 @@ describe('Arc ' + storageKeyPrefix, () => {
     const recipe = manifest.recipes[0];
     assert.isNotNull(recipe);
 
+    /*
     const foodStore = await arc.createStore(favoriteFoodClass.type.collectionOf(), undefined, 'test:1');
     assert.isNotNull(foodStore);
     recipe.handles[0].mapToStorage(foodStore);
-
+    */
```

Comment:
Just double checking, was this intentional? Do you want to re-enable the code, or just delete it?

Reply:
Delete it, I think?